src/pages/superadmin/index.tsx

```diff
@@ -93,11 +80,13 @@ export const SuperAdmin = () => {
           },
           {
             page: currentPage,
-            ...bountyStatus,
+            resetPage: true,
+            ...checkboxIdToSelectedMap,
             direction: sortOrder
           }
         );
         setBounties(bounties);
+        setTotalBounties(bounties?.length);
```

Comment:
This will break the pagination code, this is how to get the total bounties

```sh
const getTotalBounties = useCallback(async () => {
    if (startDate && endDate) {
      const totalBounties = await main.getBountiesCountByRange(String(startDate), String(endDate));
      setTotalBounties(totalBounties);
    }
  }, [main, startDate, endDate]);

```

check the master branch, for how total bounties work.

Reply:
@elraphty @ecurrencyhodler I had added this to prevent errors.  I was assuming it will remove in pagination issue or some other ticket, as it's not within our current ticket scope to handle total bounties. However I addressed the issue and also updated the [getBountiesCountByRange](https://github.com/stakwork/sphinx-tribes-frontend/pull/196/files#diff-f29d39903c159dbbc78054db5d1f97a35d4ad7726f873486d21d31a8fba7735dR2779-R2801) method to handle bounty counts for statuses. However, our backend is not providing accurate data for both getBountiesByRange and getBountiesCountByRange cases with in the selected date range. We need to fix this issue on the backend side. Despite the frontend correctly calling the endpoint, as shown in the attached video, I hope the backend issue isn't hindering the merging of this task. This task has been open for more than two weeks, and it's becoming increasingly difficult and time-consuming to merge and resolve conflicts on a daily basis.


https://github.com/stakwork/sphinx-tribes-frontend/assets/41957704/4ea7d104-cb76-4ed3-827b-57eb06439e90

Reply:
Tested the bounties count ![Image](https://i.ibb.co/vD6PMC4/Screenshot-2024-02-09-at-13-38-43.png)

Reply:
> our backend is not providing accurate data for both getBountiesByRange and getBountiesCountByRange cases with in the selected date range. We need to fix this issue on the backend side.
> 

@ecurrencyhodler Should I create an issue to address the aforementioned problem?

Reply:
@AbdulWahab3181 looks like this has been resolved in staging.

Reply:
@ecurrencyhodler I don't think so. As you can see in the attached video, I have selected the date period from February 6th to February 13th, 2024, but I am getting results from other dates when I select the status as open and assigned.

https://www.loom.com/share/0412a5a5c47b40e78347ca1e307d95ba